src/Sulu/Bundle/AdminBundle/Resources/js/containers/Sidebar/withSidebar.js

```diff
@@ -19,10 +19,6 @@ export default function withSidebar<P, C: Class<Component<P>>>(
                 super.componentDidMount();
             }
 
-            if (super.hasOwnProperty('sidebarDisposer')) {
```

Comment:
This does not longer work maybe also related to the change of extending we had AbstractFormStore